test/sampleApp/index.html

```diff
@@ -507,9 +485,38 @@ <h3>Wrapped in a <code>&lt;figure&gt;</code> element, with a <code>&lt;figcaptio
                                 var myWindow = window.open(this.href, "_blank", "width=200,height=100");	
                             }	
                         </script>
-                        
+                        <input id="change-name-btn" type="button" value="Close Curtain" onclick="changeValue()">
+                        <script>
+                          function changeValue()
+                          {
+                            var elem = document.getElementById("change-name-btn");
+                            if (elem.value=="Close Curtain") elem.value = "Open Curtain";
+                            else elem.value = "Close Curtain";
+                          }
+                        </script>
+                        <input type="reset" value="<input type=reset>">
+                        <input type="submit" value="<input disabled>" disabled>
                     </p>
                 </fieldset>
+
+                <fieldset id="clickActionSpec">
+                    <legend>ClickActionSpec</legend>
+                    <p>
```

Comment:
add an element for single click verification without nivigation

Reply:
done